source/Buoy-Collections-Tests/CollectionExtensionsTest.class.st

```diff
@@ -63,3 +63,15 @@ CollectionExtensionsTest >> testMinUsing [
 	self assert: ( #(#(3 1) #(1) #(2)) minUsing: [ :anArray | anArray first ] ) equals: 1.
 	self should: [ #() minUsing: [ :anArray | anArray first ] ] raise: Error
 ]
+
+{ #category : #'tests - copying' }
+CollectionExtensionsTest >> testWithoutFirst [
+
+	self
+		assert: #(p r u e b a) withoutFirst equals: #(r u e b a);
```

Comment:
1. The test should be changed to use #(a b c d e f) like the example, also to avoid examples in Spanish.
2. The examples in the test should be in order to explain the cases in increasing complexity.